### ISSUE TITLE
Error message with psl function

### ISSUE THREAD

**Issue by sams898** [MBoemo/DNAscent#10]
Hello Dr. Boemo,
I am excited about your new program release and the single-nucleotide resolution. I had no problems running the detect, regions, and forkSense functions, but upon trying the psl function I got the error: 
```
~/DNAscent/bin$ ./DNAscent psl -d outputDNAscent2.detect -r SAMy43AdaptersacCer3.fa -o output --threshold 0.7
Importing reference... ok.
terminate called after throwing an instance of 'std::invalid_argument'
  what():  stoi
Aborted (core dumped)
```
This error was repeated in two different files and two different computers. Do you know what might be going wrong? Thank you!

**Comment by MBoemo** [comment]
Thanks very much for reporting this - this should be resolved in [commit ab84416](https://github.com/MBoemo/DNAscent/commit/ab844162b9cb707d68714d3c58f5dc8a41b29ae0).  If you pull and recompile, all should be well.

**Comment by sams898** [comment]
Thank you so much, that resolved it!
